rosdep/base.yaml

```
@@ -1711,6 +1711,7 @@ gurumdds-2.8:
     bionic: [gurumdds-2.8]
     focal: [gurumdds-2.8]
     xenial: [gurumdds-2.8]
+    jammy: [gurumdds-2.8]
```

```suggestion
    bionic: [gurumdds-2.8]
    focal: [gurumdds-2.8]
    jammy: [gurumdds-2.8]
    xenial: [gurumdds-2.8]
```

Fix lexicographical sorting.

---

@nuclearsandwich I've done.